templates/monorepo/packages/react/package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "@mono/react",
+  "version": "0.0.0",
+  "description": "Official React library for @mono",
+  "author": "Jared Palmer",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mono/mono.git",
+    "directory": "packages/react"
+  },
+  "scripts": {
+    "start": "tsdx watch --tsconfig tsconfig.build.json --verbose --noClean",
+    "build": "tsdx build --tsconfig tsconfig.build.json",
+    "test": "tsdx test",
+    "lint": "tsdx lint",
+    "prepublish": "npm run build"
+  },
+  "main": "dist/index.js",
+  "module": "dist/react.esm.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "README.md",
+    "dist"
+  ],
+  "peerDependencies": {
+    "react": "^16.8.0",
+    "react-dom": "^16.8.0"
+  },
```

```suggestion
  "peerDependencies": {
    "react": ">=16",
  },
```

Let's keep this consistent with [the React templates]( https://github.com/formium/tsdx/blob/5e5c3f804e68609083be978a080ab905dc7ec19d/src/templates/react.ts#L17-L19)

---

Oof I didn't realize this was still there. Packages that use hooks actually need to specify React as peer dep as `"react": "^16.8.0"` (when hooks were introduced). We should discuss in another issue tho as we will need to change the other templates too